Cilsil/Cil/Parsers/ThrowParser.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Mono.Cecil.Cil;
+using Cilsil.Sil;
+using Cilsil.Sil.Expressions;
+using Cilsil.Sil.Instructions;
+using Cilsil.Sil.Types;
+using Cilsil.Utils;
+
+namespace Cilsil.Cil.Parsers
+{
+    internal class ThrowParser : InstructionParser
+    {
+        protected override bool ParseCilInstructionInternal(Instruction instruction,
+                                                            ProgramState state)
+        {
+            switch (instruction.OpCode.Code)
+            {
+                case Code.Throw:
+                    var retType = state.Method.ReturnType.GetElementType();
+                    var retNode = new StatementNode(state.CurrentLocation,
```

Comment:
Why this node never used or added to CFG?

Reply:
Forgot -- removed it thanks!